imageresizerapp/tools/chocolateyUninstall.ps1

```diff
@@ -0,0 +1,5 @@
+Uninstall-ChocolateyPackage `
+  -PackageName 'imageresizerapp' `
+  -FileType 'EXE' `
+  -Silent '/quiet /norestart /uninstall' `
```

Comment:
If I recall correctly, and I might be wrong (we should test it)... Chocolatey does not like the uninstaller path and parameters together. For most `UninstallString`s, you have to chop them apart and put them back together across the `File` and `SilentArgs` parameters.

I do wonder, however, if we shouldn't use the `QuietUninstallString` (is this standard for all apps?). Something like

```powershell
$uninstall_params = (Get-UninstallRegistryKey 'Image Resizer for Windows').QuietUninstallString -split '"'

Uninstall-ChocolateyPackage `
  -PackageName 'imageresizerapp' `
  -FileType 'EXE' `
  -Silent $uninstall_params[2] `
  -File $uninstall_params[1]
```